site/content/docs/5.2/content/reboot.md

```diff
@@ -11,6 +11,11 @@ aliases:
 toc: true
 ---
 
+<!-- Boosted mod : design callout -->
+{{< ods-incompatibility-alert >}}
+This section is about how Boosted technically works. Therefore, all the examples below are not relevant regarding the Orange Design System.
```

Comment:
Wut?? I don't understand this callout TBH.

Reply:
Initially, I added a callout design message in the section #Horizontal rules, because it was the only place where the design was not OK for ODS.
It's text was:
>These `<hr>` 3rd and 4th variants can not be used because they do not respect the Orange Design System specifications. For the 3rd variant, it is because of its opacity. For the 4th variant, it is because of its height and its opacity.
This was changed after the 1st review.
=> Shall I go back to the 1st version and place of this callout message ?

Reply:
Resolved ?